src/components/Button.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class Button extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    const { buttonName } = this.props;
+    return (
+      <div className="App">
+        <p>
+          result is:
+          { buttonName }
+        </p>
+      </div>
+    );
+  }
+}
```

Comment:
[Optional]

The `Button` component can be a function component since it is stateless.
I would recommend we render only a `button` element. The following should be enough.

```jsx
const Button = ({ name }) => (<button type='button'>{ name }</button>);
```